book/listings/todo_app/3/style.css

```diff
@@ -1,3 +1,3 @@
-.todo_row {
+box.todo_row {
   border: 1px solid @borders;
```

Comment:
FWIW there's a standard style class for this - `.frame`.

Reply:
Good to know.

I will first use css_name as suggested by @bilelmoussaoui to show this concept.
Then I will show that the same can be achieved by adding the `frame` style class.